src/go/cmd/update-collection-v3/migrations/fluentd-logs-configs/migrate.go

```diff
@@ -0,0 +1,42 @@
+package fluentdlogsconfigs
+
+import (
+	"bytes"
+	"fmt"
+
+	"gopkg.in/yaml.v3"
+)
+
+func Migrate(yamlV2 string) (yamlV3 string, err error) {
```

Comment:
I don't mind that much, but @astencel-sumo is going to complain that these should be `input` instead of `V2` and `output` instead of `V3` everywhere in this PR. :dagger: